app/org/sagebionetworks/bridge/StudyConsentBootstrapper.java

```diff
@@ -37,6 +37,7 @@ public StudyConsentBootstrapper(StudyService studyService, StudyConsentDao study
             study.setMinAgeOfConsent(18);
             study.setResearcherRole("api_researcher");
             study.setConsentNotificationEmail("bridge-testing+consent@sagebridge.org");
+            study.setSupportEmail("bridge-testing+support@sagebase.org");
```

Comment:
It looks like the emails are still coming from support@sagebridge.org. So either Eric's tests are using a cached version of the study (either cached in Redis or DDB still has the old email), or Eric's tests don't go through SES at all.